nyctaxi/taxidata/README.md

```diff
@@ -1,67 +1,100 @@
-A little tool to ingest [NYC Taxi Data](https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page) to 
-a pulsar topic to serve as an example dataset.
+The `taxidata` tool supports ingesting green or yellow NYC Taxi data to a Pulsar topic to serve as an example dataset. To obtain a data URL, right-click the data record and choose **Copy Link Address**.
+
+For more details about the dataset, see [TLC Trip Record Data](https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page).
 
 ## Setup
-Prepare a working pulsar cluster following the guide from official site: https://pulsar.apache.org/
 
-Then, clone this repository and build the `taxidata` tool:
-```bash
-go build
-```
+- Prepare a working Pulsar cluster. For details about how to create a Pulsar cluster through the `pulsar-admin` tool, see [Create Pulsar Cluster](https://pulsar.apache.org/docs/en/pulsar-admin/#create).
```

Comment:
> Since this example is designed to be used with SN Cloud, shouldn't we point people to the SN Cloud documentation here?

Thanks. I will update the link to the SN cloud documentation. Thanks